packages/components/src/components/as-category-widget/as-category-widget.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../styles/src/core/_dev';
+
+$bar--height: get-spacing(1);
+
+as-category-widget {
+  display: block;
+
+  @include padding(4);
```

Comment:
do we want to always have padding or do we want the container element to define this padding?

imagine we are placing this inside a `floating-panel` who should be the responsible of this padding? the category widget or the floating panel should define a padding for its content?

Reply:
Yes, that's a matter to discuss, because if you want to use it as a standalone component the padding would be needed, right? 🤷‍♂️

Reply:
right! IMO the component itself should not define any padding/margin...etc IMO that's responsibility of the container/user